platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -150,23 +149,38 @@ open class FeatureFragment : Fragment() {
                 foregroundResources.icons.getValue(actionButton.action)
             )
             buttonView.setOnClickListener {
-                if (actionButton == ActionButton.CLOSE)
+                if (actionButton == ActionButton.CLOSE) {
                     navigateBack()
-                else
+                } else {
                     featureContainer.triggerNavAction(actionButton.action.id)
+                }
             }
         }
     }
 
     @Suppress("unused")
     private fun handleError(error: String) {
+        if (errorDialog.isShowing || context == null) return
+
         val featureErrorMessage = context?.getString(
             R.string.message_feature_error,
             feature.name,
             error
         )
-        if (errorDialog.isShowing) return
         errorDialog.setMessage(featureErrorMessage)
+        errorDialog.setButton(
+            Dialog.BUTTON_POSITIVE,
+            context?.getString(R.string.button_feature_error_close)
```

Comment:
they can still close the dialog and not consent to the report upload. (there was no more specification about this)

Reply:
I mean there's no requirement for new UI/UX. So mainly it's the same window (as screenshot shows) with a new text button for the awareness of the user. But if they close the dialog there's no upload of the report (aka no consent) which counts as a Cancel button for us (at least for now)

Reply:
Close the dialog by tapping outside of it? This is for advanced users :).
I understand that there is no UX requirement, but adding a Cancel button does not require much effort and it would a welcomed addition.

Reply:
😄 I don't really agree that it requires advanced users. but, will give it a go anw. thanks!

Reply:
:) I don't think that iOS by default will dismiss the alert if user taps outside of it btw.